README.md

```diff
@@ -36,19 +36,65 @@ scrape_configs:
     metrics_path: /export
     params:
       command: [check_load] # Run the check_load command.
+    relabel_configs:
+      - source_labels: [__address__]
+        target_label: __param_target
+      - source_labels: [__param_target]
+        target_label: instance
+      - target_label: __address__
+        replacement: 127.0.0.1:9275 # Nrpe exporter.
     static_configs:
       - targets: # Targets to run the specified command against.
         - '127.0.0.1:5666'
         - 'example.com:5666'
+
+  - job_name: nrpe_check_apt
+    params:
+      command: [check_apt]
+      ssl: [true]
+    scrape_interval: 2m
+    scrape_timeout: 60s # this command could take a long time to execute
+    metrics_path: /export
     relabel_configs:
       - source_labels: [__address__]
         target_label: __param_target
       - source_labels: [__param_target]
         target_label: instance
       - target_label: __address__
         replacement: 127.0.0.1:9275 # Nrpe exporter.
+    static_configs:
+      - targets:
+        - example.com:5666
+
+```
+
+Result Codes in command_status:
+```
+    StatusOK       = 0
+    StatusWarning  = 1
+    StatusCritical = 2
+    StatusUnknown  = 3
 
 ```
+Sample Alert Rule:
+```
+
+groups:
+- name: Apt Status
+  rules:
+  - alert: AptUpdatesNeeded
+      # need to be explicit about looking back in time. Default is very short.
+      # choose a time that is a little longer than the scrape_interval
+    expr: command_status{job="nrpe_check_apt"} > 0
+    for: 1m
```

Comment:
A for of at least 5m is advised in general. Also https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0